shlr/Makefile

```diff
@@ -161,44 +161,26 @@ sdb-sync sync-sdb:
 	git add $I/sdb*
 	git add sdb
 
-$(TCCLIB) libr_tcc/libr_tcc.a:
-	cd libr_tcc ; $(MAKE)
-
 udis-sync udis86-sync sync-udis sync-udis86:
 	cd udis86 ; $(MAKE) sync
 
-tcc-clean tccclean:
-	cd libr_tcc ; $(MAKE) clean
-
-CFILES=i386-asm.c i386-gen.c libtcc.c tccasm.c tccelf.c tccgen.c tccpp.c
-HFILES=tcc.h i386-asm.h tcclib.h tcctok.h stab.h
+CFILES=i386-asm.c i386-gen.c
```

Comment:
well I dont know what files are related to tcc and what files are not .. except those that have tcc in the file name ofc